doc/src/Build_extras.rst

```diff
@@ -421,9 +421,9 @@ minutes to hours) to build.  Of course you only need to do that once.)
       cmake build system.  The ``lib/kim/Install.py`` script supports a
       ``CMAKE`` environment variable if the cmake executable is named other
       than ``cmake`` on your system.  Additional environment variables may be
-      provided on the command line for use by cmake.  For example, to use the
+      provided on the command-line for use by cmake.  For example, to use the
```

Comment:
I thought the hyphen was only if it was an adjective. Could be wrong though.

Reply:
I have not found an authoritative statement on this. I would not be the first inconsistency in english grammar or spelling :wink:. Feel free to make corrections as you see fit.

For several cases like this, I have reformulated the text to not use 'command-line' or 'command line'. I'll do the same for this.